test/unit/renderer/viz/expressions/binary.test.js

```diff
@@ -75,134 +76,496 @@ describe('src/renderer/viz/expressions/binary', () => {
         });
     });
 
-    describe('eval', () => {
-        describe('and', () => {
-            test('and', s.TRUE, s.TRUE, 1);
-            test('and', s.TRUE, s.FALSE, 0);
-            test('and', s.FALSE, s.FALSE, 0);
-            test('and', 0.5, s.TRUE, 0.5);
-            test('and', 0.5, 0.5, 0.25);
+    describe('.value', () => {
+        function testValue (fn, param1, param2, expected) {
+            it(`${fn}(${param1}, ${param2}) should return ${expected}`, () => {
+                const actual = s[fn](param1, param2).value;
+                expect(actual).toEqual(expected);
+            });
+        }
+
+        describe('mul', () => {
```

Comment:
Shouldn't we split also the tests into different files? Especially now that we have 1 file per operation.

Reply:
Ok, I agree, I'll change that :)